docs/getting-started-guides/vagrant.md

```diff
@@ -14,10 +14,12 @@ Running Apollo with Vagrant (and Virtualbox) is an easy way to run/test/develop
 #### Download Apollo
 
 ##### Install from source at head
-1. ```git clone https://github.com/Capgemini/apollo.git```
-2. ```cd apollo```
+1. ```git clone https://github.com/Capgemini/Apollo.git```
+2. ```cd Apollo```
 3. ```pip install -r requirements.txt```
 
+_Note: you need to run the pip command with root priviledges (e.g. sudo)_
```

Comment:
I guess this depends on your OS and what the perms are for the folder, might be worth putting in a ``` you *may*``` -